mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -4843,6 +4847,36 @@ void affixBlockwiseCopyAttributes(
   bop->setAttr("bound", b.getArrayAttr(blockwiseCopyBoundsAttr));
 }
 
+template <typename T, typename U>
+void affixGridwiseGemmAttributes(T &convOp, U &gop, OpBuilder &b) {
```

Comment:
This looks like a clean way to do it!

At the same time we need more testing to make sure those ops does transfer expected attributes as we expect them to.